assets/js/blocks/featured-category/block.js

```diff
@@ -211,7 +235,8 @@ const FeaturedCategory = ( {
 			style.backgroundColor = overlayColor.color;
 		}
 		if ( focalPoint ) {
-			style.backgroundPosition = `${ focalPoint.x * 100 }% ${ focalPoint.y * 100 }%`;
+			style.backgroundPosition = `${ focalPoint.x * 100 }% ${ focalPoint.y *
+				100 }%`;
```

Comment:
This is a bit funky.  I realize it probably satisfies linting but I'd rather see it either broken up or all pushed to the next line?

Reply:
Agree with you. I refactored this a bit so we avoid making operations inside the template literal: c26de1b9ffcba6813e36338c0dd17bcf06bc408e. This way it fits in the same line.